src/index.html

```diff
@@ -39,7 +43,22 @@ <h2 class="quiz-card__heading">Question:</h2>
       <li>Neuefische</li>
     </ul>
   </section>
-
 </main>
+<footer class="menu">
+  <nav class="menu__nav">
+    <a class="menu__nav__link" href="">
```

Comment:
The correct use of BEM would be the following and I would also add an `aria-label` and/or a `alt`-text in the image so that the user knows, where to go.
```suggestion
    <a class="menu__nav-link" href="" aria-label="Home">
```